### ISSUE TITLE
Napple Tale - Arsia in Daydream

### ISSUE THREAD

**Issue by wicksand420** [Rinnegatamante/flycast-vita-Compatibility#166]
# Game summary
- Game name: Napple Tale - Arsia in Daydream
- Game version: cdi JP

# flycast summary
- Version: v.0.1

# Issues

Stuck at loading screen on normal core, SMC needed to play the game, still it freezes at a white screen when you start the game.
# Screenshots
![](https://?)

# Recommended labels

- Crash

**Comment by Rinnegatamante** [comment]
Even with Accuracy?

**Comment by wicksand420** [comment]
I'll check again, but I think I did try all 3 last night, you have to sit through a lot of text to get to the game

**Comment by wicksand420** [comment]
Yeah, as soon as you get through the storyline it freezes at a blank white screen with SMC (accuracy) also

**Comment by wicksand420** [comment]
Gdi version works with the normal core

**Comment by Rinnegatamante** [comment]
Label?

**Comment by wicksand420** [comment]
Ingame (High)
